external.yml

```diff
@@ -72,3 +72,17 @@
         become: true
         ignore_errors: true
         when: external_network_vlan_id is not defined
+
+      - name: get default route
```

Comment:
Will we not run post.yml for OSP 17? Why has this code https://github.com/redhat-performance/jetpack/blob/master/post.yml#L6-L18 been added in this external.yml playbook again?

Reply:
In OSP17  overcloud deployment  fails if the masqurade rule is not added

Reply:
Shouldn't line 81 and line 88 in this file be
`when: external_network_vlan_id is defined  and osp_release < 17`
then?